### ISSUE TITLE
Cannot load list of indexes

### ISSUE THREAD

**Issue by zhugw** [Studio3T/robomongo#1454]
very strange when  I click `indexes` of a collection -- data_02, it gives me this error info

```
Cannot load list of indexes.

Error:
Wrong type for field (background) double != 8
```

but click `indexes` of other collections is OK.

In addition if I execute getIndexes command explicitly, it is also OK

```
> db.data_02.getIndexes()
```

**Comment by zhugw** [comment]
Maybe it's caused by this reason , exists invalid value for `background` key, e.g

```
	{
		"v" : 1,
		"key" : {
			"importStatus" : 1
		},
		"name" : "importStatus_1",
		"ns" : "foo.data_02",
		"background" : 1 # it should be 'true' not 1
	},
```

**Comment by liitfr** [comment]
Hi ! Same problem here with sparse : 

```
    {
        "v" : 1,
        "unique" : true,
        "key" : {
            "services.twitter.id" : 1
        },
        "name" : "services.twitter.id_1",
        "ns" : "meteor.users",
        "sparse" : 1
    }
]
```

**Comment by simsekgokhan** [comment]
Hi @zhugw @liitfr, thanks for reporting the problem. 
Both of the issues seem to have the same root cause in the code. 
We plan to include the fix with in the next release Robo 3T 1.2.

**Comment by simsekgokhan** [comment]
Feel free to re-open if you still have the problem with latest version.
